database_interaction.py

```diff
@@ -0,0 +1,12 @@
+import User
```

Comment:
please unify file naming convention
you either go all snake case or all capitalized

Reply:
ok  i fixed it